ui/pages/confirmations/components/gas-timing/gas-timing.component.js

```diff
@@ -113,13 +111,14 @@ export default function GasTiming({
     gasWarnings?.maxFee === GAS_FORM_ERRORS.MAX_FEE_TOO_LOW
   ) {
     return (
-      <Typography
-        variant={TypographyVariant.H7}
+      <Text
+        variant={TextVariant.bodySm}
         fontWeight={FontWeight.Bold}
+        color={TextColor.textAlternative}
```

Comment:
Instead of changing color via the CSS we are doing using the style utility props